devtools/Makefile

```diff
@@ -25,3 +25,23 @@ uninstall:
 	rm ${TOPDIR}/devtools/bin/td-layout-config
 	rm ${TOPDIR}/devtools/bin/test-runner-server
 
+tools-init:
+	docker build -t tools dev_container
+
+afl: tools-init
+	docker run -t --rm -v ${TOPDIR}:/td-shim -w /td-shim tools bash sh_script/fuzzing.sh afl
```

Comment:
Does this actually run the afl fuzz tests?
If so, should we explicitly name it as fuzz tests?

Reply:
Done